src/core/mantle/evaluator/simple/expression.ts

```diff
@@ -0,0 +1,900 @@
+import { defined, Math as CesiumMath } from "cesium";
```

Comment:
Is there any way you can split this file (expresssion.ts) in to multiple files? With over 700 lines I'm really hoping you can break it up in to smaller pieces somehow.

Reply:
I can, but I was thinking its better to have everything associated to the evaluation part together as the components inside this file aren't gonna be used anywhere else, what do you think?